src/dmangle.d

```diff
@@ -862,3 +862,32 @@ extern (C++) void mangleToBuffer(Expression e, OutBuffer* buf)
     scope Mangler v = new Mangler(buf);
     e.accept(v);
 }
+
+/******************************************************************************
+ * Mangle function signatures ('this' qualifier, and parameter types)
+ * to check conflicts in function overloads.
+ * It's different from fd.type.deco. For example, fd.type.deco would be null
+ * if fd is an auto function.
+ */
+void mangleToFuncSignature(OutBuffer* buf, FuncDeclaration fd)
+{
+    assert(fd.type.ty == Tfunction);
+    auto tf = cast(TypeFunction)fd.type;
+
+    scope Mangler v = new Mangler(buf);
+
+    //if (fd.needThis() || fd.isNested())
+    //    buf.writeByte(Type.needThisPrefix());
+    MODtoDecoBuffer(buf, tf.mod);
```

Comment:
Huh, for some reason I thought this was only for non-member functions, and member functions were checked elsewhere.